config/enjin-platform.php

```diff
@@ -230,6 +230,8 @@
         'adapter' => PlatformQrAdapter::class,
         'size' => env('QR_CODE_SIZE', 512),
         'format' => env('QR_CODE_FORMAT', 'png'),
+        'image' => env('QR_CODE_IMAGE_URL', __DIR__ . '/../resources/images/enjin-logo.jpg'),
```

Comment:
Please could we make this optional, so self-hosted users can choose not to have an image?

Reply:
We need to add this in our .env yeah?